tests/installation/multipath.pm

```diff
@@ -19,7 +19,12 @@ use testapi;
 sub run {
     # Sometimes multipath detection takes longer
     assert_screen "enable-multipath", 60;
-    send_key "alt-y";
+    if (get_var("MULTIPATH_CONFIRM") =~ /\bNO\b/i) {
+        send_key "alt-n";
+    }
+    else {
+        send_key "alt-y";
+    }
```

Comment:
Just use the ternary operator:

```perl
send_key (get_var("MULTIPATH_CONFIRM") =~ /\bNO\b/i) ? 'alt-n' : 'alt-y';
```

Reply:
Ah, of course. To be done in the upcoming amended commit ( as send_key ( ... );).